File: www/bin/hello.py
#!/usr/bin/env python3

print("Content-Type: text/html\r")
print("\r")
print("<html>")
print("<head><title>CGI Test</title></head>")
print("<body>")
print("<h1>Hello from CGI!</h1>")
print("<p>This is a Python CGI script working correctly.</p>")
print("</body>")
print("</html>")
for i in range(0, 500000):
    print('a7777', end="\n")

File: www/bin/hello.php
<?php
#!/usr/bin/env php
header("Content-Type: text/html");

echo "<html>\n";
echo "<head><title>CGI Test</title></head>\n";
echo "<body>\n";
echo "<h1>Hello from CGI!</h1>\n";
echo "<p>This is a PHP CGI script working correctly.</p>\n";

echo "<h2>For Loop Output:</h2>\n";
echo "<ul>\n";
for ($i = 1; $i <= 5000000; $i++) {
    echo "<li>Loop iteration: $i</li>\n";
}
echo "</ul>\n";

echo "<h2>Dynamic Content:</h2>\n";
echo "<p>Current timestamp: " . date('Y-m-d H:i:s') . "</p>\n";

echo "</body>\n";
echo "a7777\n";
echo "</html>\n";
?>
